devops/geoapi-workers/docker-compose.yml

```diff
@@ -18,7 +18,12 @@ services:
       driver: syslog
       options:
         tag: geoapi_workers
-    command: "celery -A geoapi.celery_app worker -l info"
+    command:  >
+      sh -c '
+      celery -A geoapi.celery_app worker -l info -Q default -n default_worker@geoapi &
+      celery -A geoapi.celery_app worker -l info -Q heavy --concurrency=6 -n heavy_worker@geoapi &
+      wait
```

Comment:
Just curious - is this preferable over letting the process hang on the `celery` command?

Reply:
@rstijerina  as we have two celery tasks, at least the first one needed to be ran in the background, `&` but second one could have been run in the foreground and then the wait skipped.
i didn't see a way to run celery and say, i want X concurrency on this queue and Y concurrency on this other queue